ledger/model/contract_execution_result.go

```diff
@@ -0,0 +1,74 @@
+package model
+
+import (
+	"fmt"
+
+	"github.com/scalar-labs/dl"
+	"github.com/scalar-labs/dl/ledger/asset"
+)
+
+// ContractExecutionResult defines the result of a contract execution.
+// It contains the result of the contract execution along with a list of asset proofs from Ledger and Auditor.
+type ContractExecutionResult struct {
+	Result        dl.JSONObject
+	Proofs        []asset.Proof
+	AuditorProofs []asset.Proof
+}
+
+// Equal checks if two contract execution results have the same values.
+func (r ContractExecutionResult) Equal(another ContractExecutionResult) (equal bool) {
+	var (
+		myResult      dl.JSONObject = r.Result
```

Comment:
I think `my` and `another` should be `ledger` and `auditor` for more clarity.

Reply:
@feeblefakie 
Sorry for confusing you.

This function is similar to 
https://github.com/scalar-labs/scalar/blob/master/common/src/main/java/com/scalar/dl/ledger/model/ContractExecutionResult.java#L79

to check if the two contract execution results are the same.